collection_requirements.rst

```diff
@@ -262,11 +262,19 @@ ways:
                <https://opensource.org/licenses/BSD-2-Clause>`_ license to make it possible for
                third-party modules which are licensed incompatibly with the GPLv3 to use them.
                Please consider this use case when licensing your own ``module_utils``.
-:All other code: All other code must be under the `GPL-3.0-or-later
-                 <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.  These plugins are run
-                 inside of the Ansible controller process which is licensed under the GPLv3+ and
-                 often must import code from the controller.  For these reasons, the GPLv3+ must be
-                 used.
+:All other code in ``plugins/``: All other code in ``plugins/`` must be under the `GPL-3.0-or-later
+                                 <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.  These plugins
+                                 are run inside of the Ansible controller process which is licensed under
+                                 the ``GPL-3.0-or-later`` and often must import code from the controller.
+                                 For these reasons, the GPLv3+ must be used.
+:All other code: Code outside ``plugins/``, in particular unit tests, may be licensed under
```

Comment:
This section is long enough that I'm concerned someone will see the "in particular unit tests" part and not notice the caveat at the end regarding unit tests. Perhaps it would be adequate to remove the "in particular unit tests" part of the sentence?

Reply:
Or perhaps unit tests should have their own section ahead of "all other code" so we can more clearly address the topic there without making this section longer.

Reply:
How about rewriting this section as
```
:All other code: Code outside ``plugins/`` may be licensed under another free software license that is compatible
                 with the `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_,
                 provided that such code does not import any other code that is licensed under
                 the ``GPL-3.0-or-later``. If the file does import other ``GPL-3.0-or-later`` code,
                 then it must similarly be licensed under ``GPL-3.0-or-later``. Note that this applies in
                 particular to unit tests; these often import code from ansible-core, plugins, module utils,
                 or modules, and such code is often licensed under ``GPL-3.0-or-later``.
```

Reply:
I think that would be adequate.

Reply:
I've added it in 626b67e21400d9670308e99b4535277662025eb7. Thanks!